docs/apache-airflow-providers-amazon/connections/athena.rst

```diff
@@ -0,0 +1,45 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/connection:athena:
+
+Amazon Athena Connection
+==========================
+
+The Athena connection type enables DBApi integrations with Athena.
+
+Authenticating to Amazon Athena
+---------------------------------
+
+Authentication may be performed using any of the authentication methods supported by `Amazon Web Services Connection <./aws.rst>`_.
+
+Default Connection IDs
+----------------------
+
+The default connection ID is ``athena_default``.
```

Comment:
Correct me if I am worng but this in fact is relevant only for athena sql does it not?
For the athena  API user needs to set aws conn id

We need to clarify this to avoid confusion. Simply by adding a note in the top of this doc and refrence to aws connection rst file for the other case

Reply:
yep, you are right! gonna fix this.